content/open-source/guides/learn-godot/beginner/index.md

```diff
@@ -0,0 +1,106 @@
+
+_Buckle your seatbelts, it going to be an awesome ride!_
+
+### Introduction: the first step in a thousand mile journey
+
+This section is for the total beginner that has little to no idea about the game industry nor the game creation process. It introduces high-level concepts that give some insights into what is to be expected.
+
+1. [GDQuest](https://www.gdquest.com/)'s very own starter guide - [Become a Game Developer: How to Learn to Code]({{< ref "tutorial/game-design/godot/become-a-game-creator/first-steps-programming/index.md" >}})
+1. [Sykoo](https://www.youtube.com/user/SykooTV/)'s [How To Get into Game Development! (Teachers, School, Self-Taught and MORE!)](https://youtu.be/EL_Le8B_iCo)
+1. [GDQuest](https://www.gdquest.com/)'s [Game Engines: what are they and how to choose the right one?](https://youtu.be/2tZK75R2K2c)
+
+### Getting started with [Godot](https://godotengine.org/) & your first actual game(s)
+
+Instead of the booooring step-by-step traditional approach of going over the theory and explaining the interface of the engine, then going through the features etc. we're going to **focus on a hands-on approach!** Best way of learning something new is through a balanced practice-theory approach. That's why we have also included a few resources that do go through the theory, but **our recommendation is to start off with the heavy hands-on resources first** and if you get stuck or if you want to understand some game engine editor & programming concepts, then go through the theory and abstract explanations.
+
+1. [GDQuest](https://www.gdquest.com/)'s [Intro to GDScript for Programming Beginners](https://youtu.be/UcdwP1Q2UlU) - **fundamentals of programming using GDScript**, the [Godot](https://godotengine.org/) scripting language. This is _mostly optional_ for people that have basic programming knowledge
+1. [GDQuest](https://www.gdquest.com/)'s [Make Your First 2D Game with Godot](https://youtu.be/Mc13Z2gboEk) & it's [associated GitHub project repository](https://github.com/GDquest/Your-First-Game-Godot-2d-Platformer) - the 1st part of **a complete free course that goes through the creation steps for a Mario platformer-like clone**
+1. [KidsCanCode](http://kidscancode.org/) [Mobile Game: Circle Jump](http://kidscancode.org/godot_recipes/games/circle_jump/) - an **end-to-end game development series** going through the process of building a one-touch abstract, but attractive mobile game
+1. [BornCG](https://www.youtube.com/user/cgboorman)'s [3D Roll the Ball playlist](https://www.youtube.com/playlist?list=PLda3VoSoc_TSBBOBYwcmlamF1UrjVtccZ) - including a tutorial series for creating a small 3D game from scratch as the 2D & 3D engines in [Godot](https://godotengine.org/) have quite a few differences between them
+1. Other useful resources:
```

Comment:
This block should be separated from the part above if it isn't part of the path. The idea of the path is to have people take each lesson one after the other.

Reply:
Forgot to mention these last comments. I've separated them through using an unordered list